app/vep/models/pipeline_model.py

```diff
@@ -1,21 +1,66 @@
 from typing import Optional, List, Literal
 from pydantic import BaseModel, validator
+from core.config import NF_COMPUTE_ENV_ID, VEP_CONFIG_INI_PATH
+
+import tempfile 
+
 class VEPConfigParams(BaseModel):
-    vcf: str
-    vep_config: str
+  vcf: str
+  vep_config: str
+
 class LaunchParams(BaseModel):
-    computeEnvId: str = "17wuWKrWOZW5JrLoGGwPdD"
-    pipeline: str
-    workDir: str
-    revision: str = "main"
-    pullLatest: bool = "true"
-    configProfiles: List[str] = ["slurmnew"]
-    paramsText: VEPConfigParams
-    preRunScript: str = "module load nextflow"
-    postRunScript: str = ""
-    stubRun: bool = "false"
-    labelIds: List[str]
-    headJobCpus: int = 1
-    headJobMemoryMb: int = 4
+  computeEnvId: str = NF_COMPUTE_ENV_ID
+  pipeline: str
+  workDir: str
+  revision: str = "main"
+  pullLatest: bool = True
+  configProfiles: List[str] = ["slurmnew"]
+  paramsText: VEPConfigParams
+  preRunScript: str = "module load nextflow"
+  postRunScript: str = ""
+  stubRun: bool = False
+  labelIds: List[str]
+  headJobCpus: int = 1
+  headJobMemoryMb: int = 4
+
 class PipelineParams(BaseModel):
-    launch: LaunchParams
+  launch: LaunchParams
+
+class ConfigIniParams(BaseModel):
+  cache: int = 1
+  dir_cache: str = "/nfs/production/flicek/ensembl/variation/data/VEP/tabixconverted/"
+  species: str = "homo_sapiens"
+  assembly: str = 'GRCh38'
+  cache_version: int = 110
+  offline: int = 1
+  force_overwrite: int = 1
+  symbol: bool = False
+  biotype: bool = False
+  transcript_version: int = 1
+  canonical: int = 1
+  ini_file: str = None
+
+  # Creates config ini file
+  def create_config_ini_file(self):
+
+    symbol = 1 if self.symbol else 0
+    biotype = 1 if self.biotype else 0
+
+    config_yaml = f'''cache {self.cache}
+dir_cache {self.dir_cache}
+species {self.species}
+assembly {self.assembly}
+cache_version {self.cache_version}
+offline {self.offline}
+force_overwrite {self.force_overwrite}
+symbol {symbol}
+biotype {biotype}
+transcript_version {self.transcript_version}
+canonical {self.canonical}
+'''
+    ini_file = tempfile.NamedTemporaryFile(prefix="vep_", suffix=".ini", dir=VEP_CONFIG_INI_PATH, delete=False)
+    try:
+      ini_file.write(config_yaml.encode())
+    finally:
+      ini_file.close()
+    self.ini_file = ini_file
```

Comment:
The ini file arguably should be a value returned from the method rather than a class property. (Not that this should block the PR; just a thought).